src/lib/receiver.js

```diff
@@ -185,6 +193,18 @@ function createReceiver (opts) {
       return rejectIncomingTransfer(transfer.id, 'no-packet')
     }
 
+    // check if the address ends with "~ipr" + our receiver id +
+    const myRequestAddressRegex = new RegExp(receiverId + '\\..+$')
```

Comment:
`receiverId` here includes `RECEIVER_ID_PREFIX` (`'~ipr.'`), so the `'.'` will be treated as a wildcard. Does that matter?